test/mocking_classes.py

```diff
@@ -652,3 +654,59 @@ class DiscreteActionConvPolicy(DiscreteActionVecPolicy):
     def _get_in_obs(self, tensordict):
         obs = tensordict.get(*self.in_keys).diagonal(0, -1, -2).squeeze()
         return obs
+
+
+class DummyModelBasedEnvBase(ModelBasedEnvBase):
+    """Dummy environnement for Model Based RL algorithms.
+
+    This class is meant to be used to test the model based environnement.
+
+    Args:
+        world_model (WorldModel): the world model to use for the environnement.
+        device (str): the device to use for the environnement.
+        dtype (torch.dtype): the dtype to use for the environnement.
+        batch_size (int): the batch size to use for the environnement.
+    """
+
+    def __init__(
+        self,
+        world_model,
+        device="cpu",
+        dtype=None,
+        batch_size=None,
+    ):
+        super(DummyModelBasedEnvBase, self).__init__(
+            world_model,
+            device=device,
+            dtype=dtype,
+            batch_size=batch_size,
+        )
+        self.action_spec = NdUnboundedContinuousTensorSpec((1,))
```

Comment:
we can delete that as it's set below in input_spec (`env._action_spec` does not exist anymore)